api/models.py

```diff
@@ -64,3 +64,18 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event (models.Model):
+
+	EVENT_STATUS = (
+		('UPCOMING', 'Upcoming events'),
+		('CURRENT', 'Todays events'),
+		('PASSED', 'Passed events'),
+	)
+
+	title = models.CharField(max_length=255, default=None)
+	start_date = models.DateTimeField()
```

Comment:
I also don't see the start time and end time, or did you think we are just meeting Wednesday and everyone should just decide for themselves when they have time to show up?